internal/provider/burn_alert_resource.go

```diff
@@ -56,6 +59,30 @@ func (*burnAlertResource) Schema(_ context.Context, _ resource.SchemaRequest, re
 					stringplanmodifier.UseStateForUnknown(),
 				},
 			},
+			"alert_type": schema.StringAttribute{
+				Optional:    true,
+				Computed:    true,
+				Description: "The alert type of this Burn Alert.",
+				Default:     stringdefault.StaticString(string(client.BurnAlertAlertTypeExhaustionTime)),
+				Validators: []validator.String{
+					stringvalidator.OneOf(helper.AsStringSlice(client.BurnAlertAlertTypes())...),
+				},
+			},
+			"budget_rate_decrease_threshold_per_million": schema.Int64Attribute{
```

Comment:
The `honeycombio_slo` resource has dropped the `per_million` in favor of allowing the author to express things as a percentage and match the authoring experience in the UI ([link](https://github.com/honeycombio/terraform-provider-honeycombio/blob/main/honeycombio/resource_slo.go#L120)).

In the same of consistency -- and arguably a nicer DX -- I think swapping this to be `budget_rate_decrease_threshold` (with or without `_percent`) makes good sense?